eoscostarica/lifebank
# Feat/notify lifebank nearby

Resolves #698
Resolves #699 

### What does this PR do?
Notify all sponsor and donor user when a new lifebank is registered near of them

### How should this be tested?
- Make run
- Register new lifebank
- Check email for user near of that lifebank coordinates
